content/docs/getting-started/create-database.md

```diff
@@ -71,6 +83,18 @@ curl -X PUT -H 'Content-Type: application/json' \
     -d '{"dbaPassword": "changeIt"}'
 ```
 
+{{< /tab >}}
+{{< tab "terraform" >}}

```

Comment:
Can you add a tab with the same name in `{{< tabs "wait-new-database" >}}` later in this section? TF will automatically wait for resources to be created and available with the default timeouts.
Otherwise clicking on Terraform will display nothing for _wait-new-database_ tabs mixin.